iroha_client/src/config.rs

```diff
@@ -66,6 +72,15 @@ impl Configuration {
                 serde_json::from_str(&proposed_transaction_ttl_ms)
                     .map_err(|e| format!("Failed to parse proposed transaction ttl: {}", e))?;
         }
+        if let Ok(transaction_status_timeout_ms) = env::var(TRANSACTION_STATUS_TIMEOUT) {
+            self.transaction_status_timeout_ms =
+                serde_json::from_str(&transaction_status_timeout_ms)
+                    .map_err(|e| format!("Failed to parse trasnasction status timeout: {}", e))?;
```

Comment:
Failed to parse trasnasction status timeout
→
Failed to parse transaction status timeout